datasets/str_sa.py

```diff
@@ -0,0 +1,80 @@
+from os import listdir
+from os.path import isfile
+
+from chipbase import Chip
+from chipbase import ChipBase
+
+
+class STR_SA(ChipBase):
+
+    def __init__(self, directory, dataset_name="STR_SA", *args, **kwargs):
+        super().__init__(dataset_name, args, kwargs)
+
+        # Get some chips
+        self.__load_chips(directory)
+
+    def __load_chips(self, directory):
+        self.chips = {}
```

Comment:
Seems like `chips` is more natural as a dictionary since each has a unique id that we use to grab it.